src/routes/landing/components/LocationSelector.js

```diff
@@ -21,7 +21,7 @@ class LocationSelector extends React.PureComponent {
 
   filter () {
```

Comment:
Maybe you could just apply the following refactor:
```jsx
filter () {
  const filterText = this.props.filterText.toLowerCase()
  if (filterText === 'wirschaffendas') {
    return locations.filter(location => !location.live)
  } else {
    return locations.filter(location => location.live)
      .filter(location => location.name.toLowerCase().includes(filterText))
  }
}
```

Reply:
Did you do it? :D

Reply:
sure :P

Reply:
@Schedulaar I'm ALWAYS applying your suggestions :D